Deepanshu_Balani/project2 tictactoe/tic-tac-toe.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
+    <title>Tic Tac Toe</title>
+    <link rel="stylesheet" type="text/css" href="tictac.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div id="tictactoe"></div>
+      <div id="player" align="center">
+        <span id="turn" align="">Player X</span>
```

Comment:
where is player O paceholder in layout ?

Reply:
it will be changed in js .
as
if (win(this)) {
    alert('Winner: Player ' + turn);
    startNewGame();
  } else if (moves === N_SIZE * N_SIZE) {
    alert('Draw');
    startNewGame();
  } else {
    turn = turn === 'X' ? 'O' : 'X';
    document.getElementById('turn').textContent = 'Player ' + turn;
  }